packages/gatsby/src/query/file-parser.ts

```diff
@@ -428,10 +468,12 @@ export default class FileParser {
     try {
       const astDefinitions =
```

Comment:
Could you tidy up this syntax while you're here. This is not very readable. tbh it's probably best to replace with a `Map` anyway